petab/calculate.py

```diff
@@ -12,6 +14,7 @@ def calculate_residuals(
         measurement_dfs: Union[List[pd.DataFrame], pd.DataFrame],
         simulation_dfs: Union[List[pd.DataFrame], pd.DataFrame],
         observable_dfs: Union[List[pd.DataFrame], pd.DataFrame],
+        parameter_df: Union[List[pd.DataFrame], pd.DataFrame],
```

Comment:
```suggestion
        parameter_dfs: Union[List[pd.DataFrame], pd.DataFrame],
```
Consistency with other arguments?

Reply:
ah right. that would depend also on if we now allow an array of parameter dfs in yaml.